samples/automl/automlTranslationDataset.js

```diff
@@ -150,41 +151,40 @@ function getDataset(projectId, computeRegion, datasetId) {
   // const datasetId = `Id of the dataset`;
 
   // Get the full path of the dataset.
-  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+  const datasetFullId = await client.datasetPath(
+    projectId,
+    computeRegion,
+    datasetId
+  );
 
   // Get complete detail of the dataset.
-  client
-    .getDataset({name: datasetFullId})
-    .then(responses => {
-      const dataset = responses[0];
-
-      // Display the dataset information.
-      console.log(`Dataset name: ${dataset.name}`);
-      console.log(`Dataset id: ${dataset.name.split(`/`).pop(-1)}`);
-      console.log(`Dataset display name: ${dataset.displayName}`);
-      console.log(`Dataset example count: ${dataset.exampleCount}`);
-      console.log(`Translation dataset specification:`);
-      console.log(
-        `\tSource language code: ${
-          dataset.translationDatasetMetadata.sourceLanguageCode
-        }`
-      );
-      console.log(
-        `\tTarget language code: ${
-          dataset.translationDatasetMetadata.targetLanguageCode
-        }`
-      );
-      console.log(`Dataset create time:`);
-      console.log(`\tseconds: ${dataset.createTime.seconds}`);
-      console.log(`\tnanos: ${dataset.createTime.nanos}`);
-    })
-    .catch(err => {
-      console.error(err);
-    });
+  const responses = await client.getDataset({name: datasetFullId});

```

Comment:
shorten this line and the next to `const [dataset] = await client.getDataset....`

Reply:
made changes as suggested!